fluentgen/pom.xml

```diff
@@ -56,6 +56,18 @@
       <version>5.9.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.13.5</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-jsr310</artifactId>
+      <version>2.13.5</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Why do we add jackson dependency here? Should we use azure-json for tests as well and not require this dependency?

Reply:
Needed as fluentgen has tests that use Azure SDK types that still depend on Jackson.